test/e2e/dash0_operator_configuration_resource.go

```diff
@@ -87,28 +86,6 @@ func deployDash0OperatorConfigurationResource(
 	}
 }
 
-func waitForAutoOperatorConfigurationResourceToBecomeAvailable() {
```

Comment:
There are no new e2e tests, but by removing these wait conditions (and `waitForManagerPodAndWebhookToStart`), a lot of the existing e2e test cases implicitly verify that a Dash0 monitoring resource can be deployed directly after running `helm install --wait`.